pipelines/toast_ground_sim.py

```diff
@@ -1510,8 +1546,9 @@ def main():
 
             if (mc == firstmc) and (ifreq == 0):
                 # For the first realization and frequency, optionally
-                # export the timestream data to a TIDAS volume.
+                # export the timestream data.
                 output_tidas(args, comm, data, totalname)
```

Comment:
These functions will probably go into the pipeline toolbox once we get to the refactor.

Reply:
Yes, the pipelines have huge amount of duplicated code that will all be ripped out and consolidated into the pipeline sub-package.  In particular the "toast_map.py" pipeline which just loads data and makes a map is not well organized.  Rather than copy and paste yet more things from the other pipelines into that one, I punted on making it better until we have done that reorganization.